deepchecks/utils/distribution/drift.py

```diff
@@ -160,9 +160,13 @@ def calc_drift_and_plot(train_column: pd.Series,
         # Should never reach here
         raise DeepchecksValueError(f'Unsupported column type for drift: {column_type}')
 
+    dist_trace_type = 'table' if dist_traces[0].type == 'table' else 'xy'
+    dist_trace_title = 'Single Value in Datasets' if dist_trace_type == 'table' else 'Distribution Plot'
```

Comment:
Title is very unclear. 
It's a statement without context, and as a user I wouldn't know what it means. I think you need to elaborate more.

Reply:
However, why even show this? If there's a single value and it's identical - why show the drift graph at all?

Reply:
if it's not identical?

Reply:
Nir suggestion:
If single values are the same doesn't show any plot.
If single values are different then show them as bars side by side

@shir22 WDYT?